packages/components/src/DateTimePickers/pickers/IncrementableScrollActionList/IncrementableScrollList/IncrementableScrollList.component.js

```diff
@@ -35,7 +47,17 @@ class IncrementableScrollList extends React.Component {
 	}
 
 	scrollRows(increment) {
-		const newRowIndex = this.state.startIndex + increment;
+		const firstIndex = 0;
+		const lastIndex = this.props.items.length - NB_ITEMS_DISPLAYED;
+		const needToScroll = increment < 0 && this.state.startIndex > firstIndex ||
+							increment > 0 && this.state.startIndex < lastIndex;
+
+		if (!needToScroll) {
+			return;
+		}
+
+		const incrementedIndex = this.state.startIndex + increment;
+		const newRowIndex = keepInBoundaries(incrementedIndex, firstIndex, lastIndex);
```

Comment:
This is always equals to `incrementedIndex` because of your `if`.
But look at the previous comment.